package-builders/install-rust.sh

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env bash
+
+set -exu -o pipefail
+
+curl \
+    --fail -sSL --connect-timeout 20 --retry 3 --output /tmp/rustup.sh \
+    "https://sh.rustup.rs"
+chmod u+x /tmp/rustup.sh
+
+/tmp/rustup.sh -y -v
```

Comment:
I’m not sure this does what you think it does. Based on reading the script, the script itself just installs rustup, not the full Rust toolchain.